atmel-samd/boards/feather_m0_adalogger/pins.h

```diff
@@ -0,0 +1,28 @@
+
+#ifndef __MICROPY_INCLUDED_ATMEL_SAMD_BOARDS_FEATHER_M0_BASIC_PINS_H__
+#define __MICROPY_INCLUDED_ATMEL_SAMD_BOARDS_FEATHER_M0_BASIC_PINS_H__
```

Comment:
Oops I missed this the first time. These header guards should be `__MICROPY_INCLUDED_ATMEL_SAMD_BOARDS_FEATHER_M0_ADALOGGER_PINS_H__` to match the filename.